README.rst_bkp

```diff
@@ -0,0 +1,345 @@
+###########
+Cyclus Core
+###########
```

Comment:
it was just to keep an easy access to old `readme` file it will be removed before merging...